src/ui/public/documentation_links/documentation_links.js

```diff
@@ -1,30 +1,32 @@
 import { metadata } from '../metadata';
 
-const urlVersion = metadata.branch;
-const baseUrl = 'https://www.elastic.co/';
+export const DOC_LINK_VERSION = metadata.branch;
+export const ELASTIC_WEBSITE_HOST = 'https://www.elastic.co/';
```

Comment:
Ah yea I guess "HOST" is officially without the scheme portion so it doesn't match up. I can rename it back.